apps/shipments/signals.py

```diff
@@ -10,9 +13,16 @@
 from .serializers import ShipmentVaultSerializer
 
 
+LOG = logging.getLogger('transmission')
+
+
 @receiver(job_update, sender=Shipment, dispatch_uid='shipment_job_update')
 def shipment_job_update(sender, message, listener, **kwargs):
+    LOG.debug(f'Shipment job update with sender {sender}.')
```

Comment:
The `sender` of this signal is just a `Shipment` class so this log isn't really informative. Maybe log `message.id` (Message.id) and/or `listener.id` (Shipment.id) instead